queue-health/graph/graph.py

```diff
@@ -85,6 +90,7 @@ def format_timedelta(delta):
 
 
 class Sampler(object):
+    mean = 0
```

Comment:
That's not completely true. This is a class member (associated to the class object).

It can have pretty terrible side effect:

```
class SomeClass(object):
    class_var = []

a = SomeClass()
b = SomeClass()

a.class_var.append(1)
print b.class_var
# print [1]
```